Source/Core/Core/ConfigManager.cpp

```diff
@@ -235,6 +235,8 @@ void SConfig::SaveCoreSettings(IniFile& ini)
   core->Set("SlotB", m_EXIDevice[1]);
   core->Set("SerialPort1", m_EXIDevice[2]);
   core->Set("BBA_MAC", m_bba_mac);
+  core->Set("BBA_SERVER", m_bba_server);
```

Comment:
New configuration values should use onion config (the stuff in Common/Config/ and Core/ConfigLoaders/) rather than SConfig, as this interface is partially deprecated.